EncryptionTests/Encryption Tests/DatastoreEncryptionTests.m

```diff
@@ -192,4 +193,15 @@ - (void)testInitReturnsNilIfEncryptionKeyProviderDoesNotReturnTheKeyUsedToOpenTh
                  @"DB can not be opened with a wrong key so the datastore can not be initialised");
 }
 
+- (void)testTextSearchIsAvailable
+{
+    CDTHelperFixedKeyProvider *provider = [CDTHelperFixedKeyProvider provider];
+    
+    CDTDatastore *datastore = [self.factory datastoreNamed:@"test_copyprovider"
+                                 withEncryptionKeyProvider:provider
+                                                     error:nil];
+    
+    XCTAssertTrue([datastore isTextSearchEnabled], @"It should be activated");
```

Comment:
@indisoluble: That bit of code tests that the `CDTDatastore+Query` category exposes all of the desired methods of the index manager to the end user.  So in your case, @gadamc is correct, it is redundant to perform both checks.